platforms/nuttx/src/canbootloader/uavcan/main.c

```diff
@@ -1185,6 +1185,18 @@ __EXPORT int main(int argc, char *argv[])
 
 	} else {
 
+		// JAKE: It is a regular boot, so we should just jump straight to the app because
```

Comment:
The commit will give the author. Please use C comments like the rest of the file

Reply:
@AlexKlimaj we can just remove all of my comments